mlflow/sklearn/__init__.py

```diff
@@ -1372,7 +1388,7 @@ def fit_mlflow(original, self, *args, **kwargs):
         return fit_output
 
     def _log_pretraining_metadata(
-        autologging_client, estimator, *args, **kwargs
```

Comment:
`args` and `kwargs` are not used in this function. additionally, they are not passed into the `_log_pretraining_metadata` for pyspark ml. 

if we pass in `args` and `kwargs` in addition to `X` and `y` we may get `X` and `y` again.

Reply:
ah sorry for the confusion. what i mean to say is that i am removing args and kwargs from sklearn's `_log_pretraining_metadata`, and that pyspark ml's `_log_pretraining_metadata` also does not use args and kwargs.